deployment/terraform/dataspace/variables.tf

```diff
@@ -35,12 +35,17 @@ variable "application_sp_object_id" {
   description = "object id of application's service principal object"
 }
 
-variable "key_file" {
-  description = "name of a file containing the private key in PEM format"
+variable "key_file_authority" {
+  description = "name of a file containing the Registration Service private key in PEM format"
```

Comment:
nit: By reading description and the variable name it gives the impression that authority and registration service are related as its usage registration service private key as per the description.

No action needed on this PR but we can discuss it later.